app/Traits/HasDatabaseStatusInfo.php

```
@@ -0,0 +1,164 @@
+    {
+        return null;
+    }
+
+    protected function showPublicUrlPlaceholder(): bool
+    {
+        return false;
+    }
+
+    public function getListeners()
+    {
+        $userId = Auth::id();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Auth' in method 'getListeners'. (undefined)

(StaticAccess)